lib/providers/services/navigation_service.dart

```diff
@@ -4,6 +4,7 @@ import 'package:flutter/material.dart';
 import 'package:provider/provider.dart';
 import 'package:seeds/screens/app/app.dart';
 import 'package:seeds/screens/app/ecosystem/dho/dho.dart';
+import 'package:seeds/screens/app/ecosystem/guardians/guardians.dart';
```

Comment:
navigation work not needed

Reply:
Some navigation classes are in there for our testing, they can be removed again later. 

But it's a bit hard to test with no UX

Reply:
Side note I tried to create unit tests for the service calls, but somehow they always fail with network errors.

In the actual app they don't fail - so IDK what it is with the unit tests but I could not get them to have a network connection and wasted about an hour on that. With no error, just failed 400 result... no error messages... I've seen this before with other network calls in unit tests in flutter / dart. I think it's an enviornment error of some kind.